### PR TITLE
Help multimodal 2

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: jina/drivers/multimodal.py [hunk 1]
```diff
@@ -9,45 +9,62 @@
...
-def _extract_doc_content(doc: 'jina_pb2.Document'):
-    # TODO discuss when do we need doc content as described in the requirement
-    # designing a driver that will extract all the required fields from the chunks of a document
-    # (buffer, blob, text, or directly embedding)
-    return doc.text or doc.buffer or (doc.blob and pb2array(doc.blob))
-
-
-def _extract_doc_embedding(doc: 'jina_pb2.Document'):
-    return (doc.embedding.buffer or None) and pb2array(doc.embedding)
+def _extract_doc_content(doc: 'jina_pb2.Document', field: str):
```
Comment (bwanglzu) [inline jina/drivers/multimodal.py:12]: we not working on emebdding anymore?

**Review by bwanglzu — APPROVED** [review]
lgtm!

**Comment by JoanFM** [comment]
@bwanglzu , the field could be embedding, we could rename the method to be more general I agree

**Comment by bwanglzu** [comment]
I'll resolve the test error in my branch

**Pull request merged**